SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -375,9 +375,25 @@ UClass* USpatialClassInfoManager::GetClassByComponentId(Worker_ComponentId Compo
 	return nullptr;
 }
 
+uint32 USpatialClassInfoManager::GetComponentIdForSpecificSubObject(const UClass& ParentClass, const UClass& SubClass) const
+{
+	const FString& SubClassPath = SubClass.GetPathName();
+	if (const FActorSchemaData* ActorSchemaData = SchemaDatabase->ActorClassPathToSchema.Find(ParentClass.GetPathName()))
+	{
+		for (const auto& Pair : ActorSchemaData->SubobjectData)
```

Comment:
I feel like there's some sort of helper function for this

Reply:
What exactly?

Reply:
Maybe you could use some kind of Find lambda here instead of for - if?
But it's not really important.

Reply:
Could some kind of a foreach somewhere but not sure on the gain.